sirepo/status.py

```diff
@@ -5,9 +5,10 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
-from pykern import pkconfig
 from pykern import pkcompat
+from pykern import pkconfig
 from pykern.pkdebug import pkdc, pkdexc, pkdlog, pkdp
+from pykern.pkunit import pkre
```

Comment:
This is not a unit test so let's not use pkunit.

Reply:
8f3de227